MinPlatformPkg/Test/Library/TestPointCheckLibNull/TestPointCheckLibNull.c

```diff
@@ -689,9 +673,9 @@ TestPointSmmReadyToLockSecureSmmCommunicationBuffer (
 **/
 EFI_STATUS
 EFIAPI
-TestPointSmmReadyToBootSmmPageProtection (
+TestPointMmReadyToBootMmPageProtection (
   VOID
-  )
+)
 {
```

Comment:
Did uncrustify move this `)` back to column 0?

In any case, please don't run uncrustify here and move it back to the original column.